lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -779,6 +813,260 @@ class ConvertAtenBmmOp : public OpConversionPattern<AtenBmmOp> {
 } // namespace
 
 namespace {
+struct QuantizationValues {
+  Value self;
+  Value zeroPoint;
+  bool isUnsigned;
+};
+
+QuantizationValues getQuantizationPerTensorValues(
+    ConversionPatternRewriter &rewriter, Location loc,
+    Aten_MakePerTensorQuantizedTensorOp makePerTensorQuantizedTensorOp,
+    const TypeConverter *const typeConverter) {
+  QuantizationValues values;
+  Value self = makePerTensorQuantizedTensorOp.getSelf();
+  Value zeroPoint = makePerTensorQuantizedTensorOp.getZeroPoint();
+  self = typeConverter->materializeTargetConversion(
+      rewriter, loc, typeConverter->convertType(self.getType()), self);
+  zeroPoint = typeConverter->materializeTargetConversion(
+      rewriter, loc, typeConverter->convertType(zeroPoint.getType()),
+      zeroPoint);
+  zeroPoint =
+      rewriter.create<arith::TruncIOp>(loc, rewriter.getI32Type(), zeroPoint);
+  auto torchDtype =
+      cast<ValueTensorType>(makePerTensorQuantizedTensorOp.getType())
+          .getDtype();
+
+  values.self = self;
+  values.zeroPoint = zeroPoint;
+  values.isUnsigned = torch_to_linalg::isUnsignedTorchType(torchDtype);
+
+  return values;
+}
+
+QuantizationValues getQuantizationPerChannelValues(
+    ConversionPatternRewriter &rewriter, Location loc,
+    Aten_MakePerChannelQuantizedTensorOp makePerChannelQuantizedTensorOp,
+    const TypeConverter *const typeConverter) {
+  QuantizationValues values;
```

Comment:
nit: move the definition down to where the members are being set at the end of the funciton.